CONTRIBUTING.md

```diff
@@ -98,6 +98,7 @@ This project uses asdf, and there is a .tool-versions file at the project root.
    `authgear.yaml` must contain the following contents for the portal to work.
 
    ```yaml
+   # yaml-language-server: $schema=../tmp/app-config.schema.json # get this by running `make export-schemas`
```

Comment:
This is not very useful. A URL pointing to github raw file of the schema would be better. But we currently do not track the JSON schema (because it is placed in a git-ignored directory).

Reply:
agree, what about below?

## Proposed

1. put at `authgear-server` root
2. put at `authgear-server/config-schemas`

```
authgear-yaml.schema.json # or keep app-config.schema.json
authgear-secrets-yaml.schema.json # or keep secrets-config.schema.json
```

Reply:
It was not done because the JSON schema was written for validation only. It was not intended to be used by a language server to facilitate autocompletion.

So the first thing is to test whether the schema at its current shape is good enough. I do not have time to verify this.

Reply:
I see. Let's not spend effort on this now then.

Will close this PR